matching-core/src/main/java/com/graphhopper/matching/LocationIndexMatch.java

```diff
@@ -52,16 +52,23 @@ public LocationIndexMatch(GraphHopperStorage graph, LocationIndexTree index) {
         this.index = index;
     }
 
-    public List<QueryResult> findNClosest(final double queryLat, final double queryLon, final EdgeFilter edgeFilter,
-                                          double gpxAccuracyInMetern) {
+    /**
+     * Returns all edges whose starting points are within the specified radius around the
```

Comment:
I'd reword it to make it clearer that there's no (general) guarantee it finds all such edges (due to searching only nine tiles).